source/docs/articles/sites/downloading-live-error-logs.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Automate Downloading Logs from the Live Environment
+description: Learn how to automate downloading logs from the Live environment of your site for debugging.
+keywords: debug, debugging sites, debug sites, debugging site, debugging mysql, debug sql, troubleshoot mysql, troubleshoot sql, database logs, db logs, where are db logs stored, where are database logs, live logs, download logs, download log, access logs, get logs
+---
+
+Logs help you find, debug, and isolate current or potential problems on your live site. You can automate the process of accessing and maintaining these logs with a simple script.
+
+## Prerequisites
+The script requires passwordless access to your codebase, and you can do this by adding an SSH key to your Pantheon user account. See [Generating SSH Keys](/docs/articles/users/generating-ssh-keys/) for more information.

```

Comment:
This isn't exactly a prerequisite. It's optional. If you don't have a SSH key added, you'll be prompted interactively for your Pantheon password.